app/mailers/user_notifier.rb

```diff
@@ -0,0 +1,9 @@
+class UserNotifier < ApplicationMailer
+  default :from => 'hipsterbites@gmail.com'
+
+  def send_signup_email(user)
+    @user = user
+    mail( :to => @user.email,
+    :subject => 'Thanks for signing up for with Hipster Bytes' )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.